dvc/commands/experiments/ls.py

```diff
@@ -1,9 +1,13 @@
 import argparse
 import logging
 
+from scmrepo.exceptions import SCMError
+
 from dvc.cli.command import CmdBase
 from dvc.cli.utils import append_doc_link
 from dvc.exceptions import InvalidArgumentError
+from dvc.repo.experiments.utils import describe
```

Comment:
This import should be lazy (so the import statement needs to be moved inside `CmdExperimentsList.run()` where we actually use `describe()`).

The benchmark CI failure is because importing `dvc.repo.experiments` here adds a lot of overhead to the default DVC initialization (we only want to import it when we are actually running an experiments command, and not on any other DVC command)

Reply:
@pmrowla Makes sense...in general what are the best practices for lazy vs eager imports? Why don't we actually use lazy imports everywhere (perhaps except for stuff that needs to be imported for every function/method/class in a module)? This is actually something I wondered about before with other projects.

Reply:
It varies from project to project, but I think the general consensus for python is usually that you should not use lazy imports until you have a specific requirement to do so (like when you need to avoid having a circular import or for specific performance reasons)